### PR TITLE
Facility Electric output variables do not show in rdd file

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
Input file using only a plant loop did not show facility electric meter report variables. This change corrects #5922. No diffs expected in example files.

Workaround is to add ElectricLoadCenter:Distribution and associated objects.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [x] Code style (parentheses padding, variable names)
 - [x] Functional code review (it has to work!)
 - [x] If defect, results of running current develop vs this branch should exhibit the fix
 - [x] CI status: all green or justified
 - [x] Performance: CI Linux results include performance check -- verify this
 - [x] Unit Test(s)
 - C++ checks:
   - [x] Argument types
   - [x] If any virtual classes, ensure virtual destructor included, other things
 - n/a IDD changes:
 - n/a If new idf included, locally check the err file and other outputs
 - n/a Documentation changes in place
 - n/a Changed docs build successfully
 - n/a ExpandObjects changes?
 - n/a If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: src/EnergyPlus/ElectricPowerServiceManager.cc [hunk 1]
```diff
@@ -116,6 +117,7 @@ namespace EnergyPlus {
 	)
 	{
 		if ( getInputFlag_ ) {
+			if ( !DataErrorTracking::AskForPlantCheckOnAbort ) return;
```
Comment (rraustad) [inline src/EnergyPlus/ElectricPowerServiceManager.cc]: First call to this function will not execute. See line 1593 of HVACManager.cc where manageElectricPowerService is called, then ManagePlantLoops is called, then AskForPlantCheckOnAbort = true and finally manageElectricPowerService is called again. This change means that only the first call of the simulation is avoided.

**Comment by rraustad** [comment]
Output variables now in rdd for the defect file:

    Output:Variable,*,Facility Total Purchased Electric Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Total Purchased Electric Energy,hourly; !- HVAC Sum [J]
    Output:Variable,*,Facility Total Surplus Electric Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Total Surplus Electric Energy,hourly; !- HVAC Sum [J]
    Output:Variable,*,Facility Net Purchased Electric Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Net Purchased Electric Energy,hourly; !- HVAC Sum [J]
    Output:Variable,*,Facility Total Building Electric Demand Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Total HVAC Electric Demand Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Total Electric Demand Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Total Produced Electric Power,hourly; !- HVAC Average [W]
    Output:Variable,*,Facility Total Produced Electric Energy,hourly; !- HVAC Sum [J]

**Comment by rraustad** [comment]
@Myoldmopar @mjwitte This looks ready to go. All 18 rdd files had the facility report variables added.

    +Output:Variable,*,Facility Total Purchased Electric Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Total Purchased Electric Energy,hourly; !- HVAC Sum [J]
    +Output:Variable,*,Facility Total Surplus Electric Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Total Surplus Electric Energy,hourly; !- HVAC Sum [J]
    +Output:Variable,*,Facility Net Purchased Electric Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Net Purchased Electric Energy,hourly; !- HVAC Sum [J]
    +Output:Variable,*,Facility Total Building Electric Demand Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Total HVAC Electric Demand Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Total Electric Demand Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Total Produced Electric Power,hourly; !- HVAC Average [W]
    +Output:Variable,*,Facility Total Produced Electric Energy,hourly; !- HVAC Sum [J]

**Comment by rraustad** [comment]
And the unit test that shows a failure runs here.
SiteGroundDomainBasement_CheckInputs_BadHorizInsExtents

**Review by mjwitte — APPROVED** [review]
Tested the defect file, all looks good.  Merged in develop and removed proper name from unit test.  Should merge after CI reports back.

**Pull request merged**